src/de/grobox/liberario/fragments/DirectionsFragment.java

```diff
@@ -390,7 +391,7 @@ private void search() {
 				pd.setCancelable(false);
 				pd.setIndeterminate(true);
 				pd.setButton(DialogInterface.BUTTON_NEGATIVE, "Cancel", new DialogInterface.OnClickListener() {
-					             @Override
+ 					             @Override
```

Comment:
Thanks, the diff looks much better now! :)
This here is the only one you missed.